lib/database-cleaner.js

```diff
@@ -88,7 +88,13 @@ var DatabaseCleaner = module.exports = function(type, config) {
   };
 
   cleaner['postgresql'] = function(db, callback) {
-    db.query("SELECT table_name FROM information_schema.tables WHERE table_schema = 'public';", function(err, tables) {
+    var schema = config.postgresql.schema || 'public';
+    var schemaPrefix = '';
+    if (schema != 'public') {
```

Comment:
Also, I think using the `set search_path` would work, but it would introduce another callback, leading to more nesting. perhaps it's a bit easier to read this way?